pkg/reconciler/pipeline/validate.go

```diff
@@ -103,7 +103,7 @@ func ValidatePipeline(pl *dfv1.Pipeline) error {
 			if s.Source.UDSource.Container == nil || s.Source.UDSource.Container.Image == "" {
 				return fmt.Errorf("invalid user-defined source vertex %q, a customized image is required", k)
 			}
-			if s.Source.HTTP != nil || s.Source.Kafka != nil || s.Source.Nats != nil || s.Source.RedisStreams != nil || s.Source.Generator != nil {
+			if s.Source.HTTP != nil || s.Source.Kafka != nil || s.Source.Nats != nil || s.Source.Generator != nil {
 				return fmt.Errorf("invalid user-defined source vertex %q, only one of 'http', 'kafka', 'nats', 'redisStreams', 'generator' and 'udSource' can be specified", k)
```

Comment:
Please also remove redisStreams from the error msg. Maybe we can do a search across the project to ensure we remove all occurrences of "redisstreams", "redis-streams", etc.

Reply:
good catch. You know it's weird, I did do:
```
grep -r -i "redis*stream" .
``` 
from the `numaflow` root but I guess I still missed it :)

Reply:
probably because I did it right before I made this particular change to line 106 and then decided I was done :)